### PR TITLE
Add links to The Hitchhiker's Guide to Logical Verification book, and Lean Together 2024

### PR DESCRIPTION
**Pull request by turibe**

### CODE REVIEW

**Comment by fpvandoorn** [comment]
I see that I made a conflicting PR to this one which got merged: #467. That makes this PR mostly obsolete (and I don't think we should be linking to Lean Together events on this page). But you also fix some typos, so maybe update this PR or make a new one? Thanks

**Comment by turibe** [comment]
Thx, I updated the PR (which just leaves some small fixes, can do a squash merge).

File: templates/learn.md [hunk 1]
```diff
@@ -4,30 +4,39 @@ There are many ways to start learning Lean, depending on your background and
...
 
 ## Books
 
-* If you prefer reading a book (with exercises), the standard mathematics oriented reference is
+If you prefer reading a book (with exercises), there are a number of freely available Lean books.
+These are available as PDFs, but usually meant to be read interactively in VSCode,
+doing Lean exercises on the fly:
+
+* The [Hitchhiker's Guide to Logical Verification (2023 Edition)](https://github.com/blanchette/logical_verification_2023) is a good introduction to both the verification and programming aspects of Lean,
+with an associated VSCode project and exercises. [PDF here](https://github.com/blanchette/logical_verification_2023/blob/main/hitchhikers_guide.pdf).
```
Comment (fpvandoorn) [inline templates/learn.md:33]: This should definitely not be the first suggestion. Also, I already added it to the list below.

File: templates/learn.md [hunk 2]
```diff
@@ -4,30 +4,39 @@ There are many ways to start learning Lean, depending on your background and
...
   [Glimpse of Lean tutorial](https://github.com/PatrickMassot/GlimpseOfLean).
   This contains four basic files covering some fundamental aspects of proving
   using Lean, and then independent topic files about elementary analysis,
   abstract topology and mathematical logic.
 
 ## Books
 
-* If you prefer reading a book (with exercises), the standard mathematics oriented reference is
+If you prefer reading a book (with exercises), there are a number of freely available Lean books.
+These are available as PDFs, but usually meant to be read interactively in VSCode,
```
Comment (fpvandoorn) [inline templates/learn.md:29]: most books will be read in html, so I'm not sure if this sentence adds much...

**Comment by fpvandoorn** [comment]
Yes, don't worry about multiple commits, I'll squash merge in the end.

File: templates/learn.md (hunk 2 above)
Comment (turibe) [reply templates/learn.md]: Adding "html or pdfs"... the important point is recommending reading in VSCode (which applies to several of the books).

File: templates/learn.md (hunk 1 above)
Comment (turibe) [reply templates/learn.md]: Fixed

**Comment by fpvandoorn** [comment]
Thanks!

**Comment by fpvandoorn** [comment]
(oh, I forgot to update the title of this PR after the changes. Oh well)

**Comment by turibe** [comment]
> Thanks for the fixes!
> 
> (oh, I used the PR title as the commit message, which is outdated. Oh well)

Let's just say that I forgot the word "remove" in the PR title :-).

**Pull request merged**